module/Application/view/layout/layout.phtml

```diff
@@ -37,7 +56,10 @@
                 <div class="collapse navbar-collapse" id="navbarSupportedContent">
                     <ul class="navbar-nav">
                         <li class="nav-item active">
-                            <a class="nav-link" href="<?= $this->url('home') ?>">Home <span class="sr-only">(current)</span></a>
+                            <a
+                                class="nav-link"
+                                href="<?= $this->url('home') ?>"
+                            >Home <span class="sr-only">(current)</span></a>
```

Comment:
`<a` and `>` on separated lines?! I don't know if you use an automatic function, but that's not nice.
```suggestion
                            <a class="nav-link" href="<?= $this->url('home') ?>">
                                Home <span class="sr-only">(current)</span>
                            </a>
```